README.md

```diff
@@ -78,3 +78,13 @@ The charts in the `Incubator` directory are those that do not meet these criteri
 ## Status of the Project
 
 This project is still under active development, so you might run into [issues](https://github.com/nuodb/nuodb-helm-charts/issues). If you do, please don't be shy about letting us know, or better yet, contribute a fix or feature.
+
+## Testing
+
+To run the tests in this project, Go 1.12 is required. This project uses GOPATH to manage dependencies. The easiest way to clone the repository and run a test suite (e.g. `test/integration`) is as follows:
+
+```
+go get github.com/nuodb/nuodb-helm-charts
+cd $(go env GOPATH)/src/github.com/nuodb/nuodb-helm-charts
+go test ./test/integration
```

Comment:
also `-v` and `-timeout 60`.

This will only run 1/3 of the tests. And skip all the minikube tests.

Reply:
What is the 60 second timeout? It definitely takes longer than 60 seconds to run the minikube tests. Also, does that command run both the short and long variants?

Reply:
the default is 15min. It needs to be bumped to 60/120 min. See https://github.com/nuodb/nuodb-helm-charts/blob/master/scripts/ci/determine_and_run_tests.sh